include/openthread/udp.h

```diff
@@ -289,6 +289,18 @@ void otUdpForwardReceive(otInstance *        aInstance,
                          const otIp6Address *aPeerAddr,
                          uint16_t            aSockPort);
 
+/**
+ * Determines if the given UDP port is used by OpenThread, or belongs entirely to the platform
```

Comment:
What does it mean for a UDP port to be `used by OpenThread`. Define it clearly in the Doxygen?